### PR TITLE
refactor: Prototype action functions

### PR DESCRIPTION
**Pull request by phorward**
This pull requests refactors the code for all prototype standard actions as follows:

- Consequent use of "key" in signature
- Clean-up kwargs check
- Clean-up "bounce" check
- Make code more readable
- Instanciate variables right before their use, and not on top

### CODE REVIEW

**Review by sveneberth — DISMISSED** [review]
LGTM, but I've some related suggestions: https://github.com/phorward/viur-core/pull/5

File: src/viur/core/prototypes/list.py
Comment (ArneGudermann) [reply src/viur/core/prototypes/list.py:117]: This is stands for all (`edit`,...)

**Review by ArneGudermann — CHANGES_REQUESTED** [review]

**Comment by phorward** [comment]
> LGTM, but I've some related suggestions: [phorward#5](https://github.com/phorward/viur-core/pull/5)

Can we please do this PR separately after merge?

**Comment by ArneGudermann** [comment]
Can you take a look at https://github.com/phorward/viur-core/pull/6. Than I'm fine with this.

**Comment by phorward** [comment]
> Can you take a look at [phorward#6](https://github.com/phorward/viur-core/pull/6). Than I'm fine with this.

No, because this ["kinda hackish" thing](https://github.com/viur-framework/viur-core/blob/main/core/request.py#L642) is the reason why no 406, but 500 is thrown.  Therefore, I won't merge it, and would like to merge this PR as a step towards a more stable version.

File: src/viur/core/prototypes/list.py
Comment (phorward) [inline src/viur/core/prototypes/list.py:117]: Turn this back.
```suggestion
    def view(self, key: db.Key | str | int | None, *args, **kwargs) -> Any:
```

**Pull request merged**
